projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "gpio.h"
+
+typedef enum {
+  LIGHTS_CONFIG_BOARD_TYPE_FRONT = 0,
+  LIGHTS_CONFIG_BOARD_TYPE_REAR,
+  NUM_LIGHTS_CONFIG_BOARDS
+} LightsConfigBoardType;
+
+typedef struct LightsConfig {
+  const LightsConfigBoardType board_type;
+  const GPIOAddress *addresses;
+  const uint8_t num_addresses;
```

Comment:
Just so you know, you don't save any space by using a u8 here. Due to struct padding, you'll end up with 24 bits of padding here and at `num_supported_events`.

Reply:
interesting 🤔 I'll read about it. 
Do I need to change something here?